src/index.html

```diff
@@ -1,17 +1,237 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
-  </head>
-  <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
-  </body>
+    <head>
+        <meta charset="UTF-8">
+        <meta
+          name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+        >
+        <meta 
+          http-equiv="X-UA-Compatible" 
+          content="ie=edge"
+        >
+        <title>HTML Form</title>
+        <link rel="stylesheet" href="./style.css">
+    </head>
+    <body>
+        <script type="text/javascript" src="./main.js"></script>
+        <form 
+          action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+          method="post"
+          class="form"
+        >
+
+            <fieldset class="group">
+                <legend>
+                  Personal information
+                </legend>
+                <div class="simple">
+                    <label for="sname">
+                      Surname:
+                        <input 
+                         type="text" 
+                         name="sname"
+                         id="sname" 
+                         required 
+                         autocomplete="off"
+                        >
+                    </label>
+                </div>
+                <div class="simple">
+                    <label for="name">
+                      Name:
+                        <input 
+                         type="text" 
+                         name="name" 
+                         required 
+                         autocomplete="off"
+                         id="name"
+                        >
+                    </label>
+                </div>
```

Comment:
Please check all places
```suggestion
                <div class="simple">
                  <label for="name">
                    Name:
                    <input 
                      type="text" 
                      name="name" 
                      required 
                      autocomplete="off"
                      id="name"
                    >
                  </label>
                </div>
```